src/patternfly/components/Card/card-title-text.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="pf-c-card__title-text{{#if card-title-text--modifier}} {{card-title-text--modifier}}{{/if}}"
+  {{#if card-title-text--attribute}}
+    {{{card-title-text--attribute}}}
+  {{/if}}>
+  {{#if card-title-text--text}}
+    {{card-title-text--text}}
+  {{else if @partial-block}}
```

Comment:
Is there a reason to add the `if @partial-block` here rather that it just being `else`? (same question in card-title.hbs)

Reply:
The reason I do this is to avoid compilation errors when partial content isn't necessary/passed to the `.hbs` partial, as well as avoiding having to close tags `{{#> thing}}{{/thing}}` as opposed to `{{> thing [props that may not be {{{card-title-text--text}}`.

This would be better written as:

```suggestion
  {{#if card-title-text--value}}
    {{{card-title-text--value}}}
  {{/if}}
  {{#if @partial-block}}
      {{> @partial-block}}
  {{/if}}
```